features/keystone.feature

```diff
@@ -0,0 +1,15 @@
+@keystone
+Feature: Keystone
+  As an administrator
+  I want to make sure the Keystone component is configured and running
+  In order to provide authentification and authorization service to  other Openstack components
+
+  Scenario: User create # was test no.30
+    Given I have a tenant named 'openstack'
+    When I can delete a user named 'test_user'
```

Comment:
Just a grammar nitpick: this should read "When I delete ...", because a "When $X" clause is not concerned with whether the $X action is possible - it already assumes it's possible.  In contrast, "Then" clauses *are* concerned with stating that the action is possible, so "Then I can ..." is correct.